lib/ansible/modules/windows/win_blockinfile.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
```

Comment:
No copyright statement ? Then you may want to hand over copyright to the Ansible project instead ?

Reply:
Sounds good to me, how do I do that?

Reply:
Simply add a copyright-statement with Ansible, just look at other modules that have Ansible as the copyright holder.